renpy.el

```diff
@@ -438,8 +438,10 @@
   ;; First avoid a sequence preceded by an odd number of backslashes.
   (syntax-propertize-rules
    (;; Backrefs don't work in syntax-propertize-rules!
-    (concat "\\(?:\\([RUru]\\)[Rr]?\\|\\(?:\\=\\|[^\\]\\)\\(?:\\\\.\\)*\\)?" ;Prefix.
-	    "\\(?:\\('\\)'\\('\\)\\|\\(?2:\"\\)\"\\(?3:\"\\)\\)")
+    (concat
+     ;Prefix.
+     "\\(?:\\([RUru]\\)[Rr]?\\|\\(?:\\=\\|[^\\]\\)\\(?:\\\\.\\)*\\)?"
+     "\\(?:\\('\\)'\\('\\)\\|\\(?2:\"\\)\"\\(?3:\"\\)\\)")
```

Comment:
The previous version of this seemed fine as it was within the fill-column, and I think the comment only relates to the first string which is being concatenated. As it is now the indentation will be wrong when reapplied.